website/docs/r/domain.html.markdown

```diff
@@ -408,10 +408,15 @@ The block supports:
 * `type` - the type of graphics emulation (default is "spice")
 * `autoport` - defaults to "yes"
 * `listen_type` - "listen type", defaults to "none"
+* `listen_address` - (Optional) IP Address where the VNC listener should be started if
+`listen_type` is set to `address`. Defaults to 127.0.0.1
```

Comment:
I was wondering if it wouldn't be better to group these `listen_*` parameters in a

```hcl
   ...
   listen {
      type = "address"
      address = "127.0.0.1"
   }
   ...
```